cpt/requirements.txt

```diff
@@ -1,3 +1,3 @@
 six>=1.10.0
-conan>=1.7.0, <1.19.0
+conan>=1.7.0, <=1.19.0
```

Comment:
@lasote Shouldn't this have been `< 1.20.0` to allow patch versions from 1.19.x?

Reply:
https://github.com/conan-io/conan-package-tools/pull/411

Reply:
anyway, good catch @Croydon ! I just saw that because a report!

Reply:
Well, my Conan was suddenly downgraded when I locally installed bincrafters-package-tools (which in turns installs conan-package-tools), so I knew something was off... 👁